Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -2287,6 +2292,11 @@ return
 					Type = "bool",
 					Notes = "Does this block fully occupy its voxel - is it a 'full' block?",
 				},
+				m_Hardness =
+				{
+					Type = "number",
+					Notes = "The block's hardness. The bigger the harder the block.",
```

Comment:
It's should be the time that it takes to mine a block, but I think as it's also handled client side, it will have visual problems.